weblate/addons/resx.py

```diff
@@ -31,8 +31,8 @@ class ResxUpdateAddon(BaseCleanupAddon):
     verbose = _("Update RESX files")
     description = _(
         "Update all translation files to match the monolingual base file. "
-        "It removes stale translation keys and adds new ones with source "
-        "string as a translation."
+        "It removes unused strings, and adds new ones wherein the translation is "
```

Comment:
Removes from where, unused as in?

Reply:
Short summary: It does pretty much same thing as msgmerge does for gettext PO files

It takes the source language file and removes from target language files any strings which are no longer present there, adds new ones and updates changed ones. This used to be part of the cleanup addon, but it turned out that only part of RESX users want the updating part, it depends how RESX is implemented in the application.